src/scss/_main.scss

```diff
@@ -1333,3 +1356,19 @@ nav {
 .overflow-auto {
   overflow: auto;
 }
+
+.flex {
```

Comment:
these are starting to accumulate. just a note that we should consider separating these generic utility classes out soon.

Reply:
good point, and there will be some more in PRs I have in flight, I'll consider this sooner rather than later in a smaller PR :)